models/articles-models.js

```diff
@@ -20,13 +20,30 @@ exports.selectArticles = (sort_by='created_at', order='desc') => {
         "comment_count"
     ]
 
+    const validTopics = [  
+        "mitch",
+        "cats",
+        "paper"
+    ]
```

Comment:
This won't work with your production database, the topics are different. 

Instead of doing this and then checking the array, you have to make a query to the database to check if the topic is there, then it will work for any database.